vllm/v1/worker/gpu_worker.py

```diff
@@ -212,6 +212,14 @@ def compile_or_warm_up_model(self) -> None:
             self.model_runner._dummy_run(size)
         if not self.model_config.enforce_eager:
             self.model_runner.capture_model()
+
+        # Warm up sampler and preallocate memory buffer for logits and other
+        # sampling related tensors of max possible shape to avoid memory
+        # fragmentation issue.
+        self.model_runner._dummy_sampler_run(
+            hidden_states=self.model_runner._dummy_run(
+                num_tokens=self.scheduler_config.max_num_seqs))
```

Comment:
```suggestion
                num_tokens=self.scheduler_config.max_num_batched_tokens))
```

Reply:
Oh I just realized that `max_num_seqs` is more accurate here. Ignore this comment.